### PR TITLE
operator percona-postgresql-operator (1.3.0)

### PR DESCRIPTION
**Pull request by cap1984**
Signed-off-by: Ivan Pylypenko <ivan.pylypenko@percona.com>

Thanks submitting your Operator. Please check below list before you create your Pull Request.

### Updates to existing Operators

* [x] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [x] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [x] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [x] Have you tested an update to your Operator when deployed via OLM?
* [x ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [x] Modify more than one operator
* [x] Modify an Operator you don't own
* [x] Rename an operator - please remove and add with a different name instead
* [x] Modify any files outside the above mentioned folders
* [x] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [x] Description about the managed Application and where to find more information
2. [x] Features and capabilities of your Operator and how to use it
3. [x] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [x] Human readable name and 1-liner description about your Operator
* [x] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [x] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [x] Links to the maintainer, source code and documentation
* [x] Example templates for all Custom Resource Definitions intended to be used
* [x] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

File: operators/percona-postgresql-operator/1.3.0/percona-postgresql-operator.v1.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  version: 1.3.0
```
Comment (mvalarh) [inline operators/percona-postgresql-operator/1.3.0/percona-postgresql-operator.v1.3.0.clusterserviceversion.yaml:233]: Please add `replaces: percona-postgresql-operator.v1.2.0`

**Review by mvalarh — CHANGES_REQUESTED** [review]

File: operators/percona-postgresql-operator/1.3.0/percona-postgresql-operator.v1.3.0.clusterserviceversion.yaml (hunk 1 above)
Comment (cap1984) [reply operators/percona-postgresql-operator/1.3.0/percona-postgresql-operator.v1.3.0.clusterserviceversion.yaml:233]: done

File: operators/percona-postgresql-operator/1.3.0/percona-postgresql-operator.v1.3.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,666 @@
...
+  name: percona-postgresql-operator.v1.3.0
+  namespace: default
+  annotations:
+    categories: Database
+    certified: 'true'
+    createdAt: '2022-08-11T13:11:44.000Z'
+    description: >-
+      Percona Operator for PostgreSQL manages the lifecycle of Percona PostgreSQL
+      cluster instances.
+    containerImage: 'percona/percona-postgresql-operator:1.3.0-postgres-operator'
```
[the remark this answers is not in the document]
Comment (cap1984) [reply operators/percona-postgresql-operator/1.3.0/percona-postgresql-operator.v1.3.0.clusterserviceversion.yaml:14]: @J0zi image host you mean?
Is it ok to add docker.io at the path start?

[the remark this answers is not in the document]
Comment (J0zi) [reply operators/percona-postgresql-operator/1.3.0/percona-postgresql-operator.v1.3.0.clusterserviceversion.yaml:14]: @cap1984 yes, exactly. Please do so. Thank you :)

[the remark this answers is not in the document]
Comment (cap1984) [reply operators/percona-postgresql-operator/1.3.0/percona-postgresql-operator.v1.3.0.clusterserviceversion.yaml]: done

**Pull request merged**